src/main/java/net/pms/newgui/Wizard.java

```diff
@@ -78,116 +77,137 @@ public static void run(final PmsConfiguration configuration) {
 			yesNoOptions[1]
 		);
 
-		if (whetherToStartMinimized == JOptionPane.YES_OPTION) {
-			configuration.setMinimized(true);
-		} else if (whetherToStartMinimized == JOptionPane.NO_OPTION) {
+		if (whetherToSelectDefaultOptions == JOptionPane.YES_OPTION || whetherToSelectDefaultOptions == JOptionPane.CLOSED_OPTION) {
 			configuration.setMinimized(false);
-		}
-
-		// Ask if their network is wired, etc.
-		int networkType = JOptionPane.showOptionDialog(
-			null,
-			Messages.getString("Wizard.7"),
-			String.format(status, currentQuestionNumber++),
-			JOptionPane.YES_NO_CANCEL_OPTION,
-			JOptionPane.QUESTION_MESSAGE,
-			null,
-			networkTypeOptions,
-			networkTypeOptions[1]
-		);
-
-		switch (networkType) {
-			case JOptionPane.YES_OPTION:
-				// Wired (Gigabit)
-				configuration.setMaximumBitrate("0");
-				configuration.setMPEG2MainSettings("Automatic (Wired)");
-				configuration.setx264ConstantRateFactor("Automatic (Wired)");
-				break;
-			case JOptionPane.NO_OPTION:
-				// Wired (100 Megabit)
-				configuration.setMaximumBitrate("90");
-				configuration.setMPEG2MainSettings("Automatic (Wired)");
-				configuration.setx264ConstantRateFactor("Automatic (Wired)");
-				break;
-			case JOptionPane.CANCEL_OPTION:
-				// Wireless
-				configuration.setMaximumBitrate("30");
-				configuration.setMPEG2MainSettings("Automatic (Wireless)");
-				configuration.setx264ConstantRateFactor("Automatic (Wireless)");
-				break;
-			default:
-				break;
-		}
-
-		// Ask if they want to hide advanced options
-		int whetherToHideAdvancedOptions = JOptionPane.showOptionDialog(
-			null,
-			Messages.getString("Wizard.11"),
-			String.format(status, currentQuestionNumber++),
-			JOptionPane.YES_NO_OPTION,
-			JOptionPane.QUESTION_MESSAGE,
-			null,
-			yesNoOptions,
-			yesNoOptions[0]
-		);
-
-		if (whetherToHideAdvancedOptions == JOptionPane.YES_OPTION) {
+			configuration.setMaximumBitrate("90");
```

Comment:
I would recommend to use `automatic maximum bandwidth` instead of this. The default setting anticipate that the connection is wired but the wireles can be used. So checking the maximum bandwidth could avoid problems users should have with sending the transcoded files exceeding the network bandwidt.

Reply:
```suggestion
			configuration.setAutomaticMaximumBitrate(true);
```